maptime/testfiles/test_file_1.go

```diff
@@ -0,0 +1,20 @@
+package testfiles
```

Comment:
could you rename dir/package to `testdata`, more [idiomatic](https://github.com/golang/go/issues/14715)